addons/kube-oidc-proxy/kube-oidc-proxy.yaml

```diff
@@ -7,8 +7,8 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: kube-oidc-proxy
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "0.1.1-4"
-    appversion.kubeaddons.mesosphere.io/kube-oidc-proxy: "v0.1.1"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.2.0-0"
```

Comment:
does `-0` work? Took a quick look at some of the other addons and their revisions usually started with `-1`. 

eg
- https://github.com/mesosphere/kubernetes-base-addons/commit/017619d052529da28e146be14550be43d6913ea4#diff-2628512f91700a796f692187c84da1abR10
- https://github.com/mesosphere/kubernetes-base-addons/commit/93acf72a1cfb0ae2d0cd4ac75af1598c1f7e1a93#diff-c27922faad5faec9771430e0b4a10f7aR9

Reply:
hmm, i thought it would, but i can change it to be safe.